content/blog/2020-01-14-maps-projected-change.md

```diff
@@ -43,4 +43,4 @@ Click on the overlay to get a popup box with data value for the underlying grid
 And if you find the maps useful, you can download the overlays as images (PNG) or as raster data (GeoTIFF) that you can import into a GIS program or other analytic environment.
 
 ## Explore
-We hope [this tool](https://cal-adapt.org/tools/maps-of-projected-change/) makes it easier to understand how climate change may impact California’s temperatures and precipitation, compared to the state’s recent climate history. If you want to learn more about how to use climate projections, check out our [guidance page](https://cal-adapt.org/resources/using-climate-projections/). And if you have any questions, don’t hesitate to reach out to the Cal-Adapt support team at [support@cal-adapt.org](mailto:support@cal-adapt.org).
+We hope [this tool](/tools/maps-of-projected-change/) makes it easier to understand how climate change may impact California’s temperatures and precipitation, compared to the state’s recent climate history. If you want to learn more about how to use climate projections, check out our [guidance page](/help/glossary/climate-projection). And if you have any questions, don’t hesitate to reach out to the Cal-Adapt support team at <support@cal-adapt.org>.
```

Comment:
Actually this link does work, and there are quite a few links to the glossary items throughout the content section, especially linking from one glossary term to another. If you search on `/help/glossary` on the directory structure you'll see all the links.

Reply:
@elehmer I'm reffering to the following link [/help/glossary/climate-projection](https://dev.cal-adapt.org/help/glossary/climate-projection)

Reply:
Oh, I see the problem but not why. This 404's true, but /help/glossary/climate-normal and /help/glossary/global-climate-model do?? And to add to the confuse the climate-projection linkage works on my local npm dev server? I can change the URLs that don't work to /help/glossary

Reply:
I'm sorry about the confusion over this. I have added onto this PR changes to all content files that refer to sub-items in the glossary and point them instead to the top level of the glossary.

Reply:
No worries! I think what's going on is that the dynamic routing is happening locally with the slugs but because we aren't linking to each slug in the app the pages for them don't get built when running `sapper export` which happens in the `npm run export` script.